packages/graphql/pubspec.yaml

```diff
@@ -1,6 +1,6 @@
 name: graphql
 description: A stand-alone GraphQL client for Dart, bringing all the features from a modern GraphQL client to one easy to use package.
-version: 5.2.0-beta.5
+version: 5.2.0-beta.6
```

Comment:
this should not be bumped by you, but by me at related time

Reply:
I have changed to ```feat(graphql)```
running make requires melos package which i do not have 
```
No active package melos.
```

regarding version please change it accordingly :)

Reply:
p.s. chaneg is rather trivial :) just introduced function which is triggered before reconnect. Thats it.

Reply:
All the PR should pass the CI before merging!

Reply:
and you should not bump the version number here otherwise you will break the CI and my release workflow

Reply:
I cannot do that. I have already told you that formatter is different. One step fails because of this. I asked you to run through your formatter because I do not have melos

Reply:
run `make dep && make`.

In flutter there is no "my formatted or your formatted" there is the formatter.

Reply:
As I already asked you pls do trivial things your self because there are more troubles in this library rather than formatting 😄 if it is sso difficult to get it merged I will skip idea to implement autoreconnect pause as we talked on email

Reply:
will do that hovever it installs melo as global dep which I am not using. I am using standart dart formatter

Reply:
melos is not the formatter, please do not start not useful fighting without any reason. the command that I suggest and push the code, just this.